### DISCUSSION TITLE
How to use text, draw, image, zoom etc functionality

### DISCUSSION THREAD

**Discussion by paritosh-mahale** [wojtekmaj/react-pdf#1839] [Q&A]
Hello everyone,

I am currently working on a project where I need to utilize various functionalities provided by PDF.js, such as text selection, drawing annotations, embedding images, and zooming in and out of PDF documents. However, I am facing difficulties finding these specific features within the React-PDF library.

Has anyone successfully integrated these PDF.js functionalities into a React application using React-PDF? If so, could you please guide me on how to achieve this? Any examples or resources would be highly appreciated.

Thank you for your assistance!

**Comment by wojtekmaj** [comment]
This isn't the aim of this project, but you might be looking for something like this: https://github.com/react-pdf-viewer/react-pdf-viewer
